maat-court-data-api/src/main/java/gov/uk/courtdata/dces/service/ConcorContributionsService.java

```diff
@@ -25,16 +23,18 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.time.LocalDate;
 import java.util.List;
 import java.util.NoSuchElementException;
 import java.util.Optional;
 import java.util.Set;
 
+import static gov.uk.courtdata.enums.ConcorContributionStatus.SENT;
```

Comment:
Personally i'm not a fan of static imports for non-test code, but I don't think there are any hard and fast style guidelines about it so it's down to personal taste.

Reply:
Completely agree with you about being cautious with the use of static imports - especially for static final constants.

However, this is an `enum` and it's the argument to the `updateConcorContributionStatus*` method in a class that's mostly to do with ConcorContributions, so given that context, I think ease-of-reading tips the balance.